app/Http/Controllers/EducationalInformationDiplomaController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\Diploma;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Session;
+
+class EducationalInformationDiplomaController extends Controller
+{
+    public function store(Request $request)
+    {
+        $request->validate([
+            'diploma_college' => 'required|min:5|max:20',
+            'diploma_university' => 'required|min:5|max:20',
+            'diploma_aggregates' => 'required|numeric|between:0,99.99',
+            'out_of' => 'required|size:4|digits:3',
+            'diploma_major' => 'required|min:5|max:20',
+            'major_title' => 'required|min:4|max:20',
+            'diploma_langauge' => 'required|min:3|max:20',
+            'diploma_year_form' => 'required|size:4|digits:4',
+            'diploma_year_to' => 'required|size:4|digits:4',
+            'completion_status' => 'required',
```

Comment:
Validations are wrong.  all max:254
out_of can be of 2 or 3 or even 1 digit. eg 1.5 out of 5
remove size and and digits as it can be grade too like A B C ... F

Reply:
Do this in all forms